.idea/codeStyleSettings.xml

```diff
@@ -16,7 +16,7 @@
             <package name="" withSubpackages="true" static="false" />
           </value>
         </option>
-        <option name="ENABLE_JAVADOC_FORMATTING" value="false" />
```

Comment:
is this not necessary? I found that reformatting questions would really screw up the //question_comment sections.